From 836d4d0856ddbe853faed66df2a14b979e079a18 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Mon, 2 Jul 2018 23:37:33 +0200
Subject: [PATCH 5/5] Shorten build id

---
 core/build_id.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index 7f64a832c..8998a55ab 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=OPM8.190505.001
+export BUILD_ID=OPM8.190505
-- 
2.17.1

