From 75049489b87577d28950ab76c7d6db1d5e05cb12 Mon Sep 17 00:00:00 2001
From: svoboda18 <ruspam18@gmail.com>
Date: Thu, 7 Feb 2019 22:00:34 +0000
Subject: [PATCH] Fix ussd on mtk ril

Change-Id: Ia70296794a553a3692e8415a6d1fa62cc2535e13

diff --git a/src/java/com/android/internal/telephony/GsmCdmaPhone.java b/src/java/com/android/internal/telephony/GsmCdmaPhone.java
index adfb2f2fb..2b83f0337 100644
--- a/src/java/com/android/internal/telephony/GsmCdmaPhone.java
+++ b/src/java/com/android/internal/telephony/GsmCdmaPhone.java
@@ -2057,7 +2057,8 @@ public class GsmCdmaPhone extends Phone {
             // Complete pending USSD
 
             if (isUssdRelease) {
-                found.onUssdRelease();
+                //found.onUssdRelease();
+		found.onUssdFinished(ussdMessage, isUssdRequest);
             } else if (isUssdError) {
                 found.onUssdFinishedError();
             } else {
-- 
2.17.1

