From b47091748618dfcb84e3d7db0a6cb8cd57218dc6 Mon Sep 17 00:00:00 2001
From: svoboda18 <ruspam18@gmail.com>
Date: Fri, 8 Mar 2019 17:43:54 +0000
Subject: [PATCH] mediatek: fix some mtk ril bugs


diff --git a/src/java/com/android/internal/telephony/GsmCdmaPhone.java b/src/java/com/android/internal/telephony/GsmCdmaPhone.java
index adfb2f2..cc115fd 100644
--- a/src/java/com/android/internal/telephony/GsmCdmaPhone.java
+++ b/src/java/com/android/internal/telephony/GsmCdmaPhone.java
@@ -2057,7 +2057,11 @@ public class GsmCdmaPhone extends Phone {
             // Complete pending USSD
 
             if (isUssdRelease) {
-                found.onUssdRelease();
+                if (SystemProperties.getBoolean("persist.sys.radio.ussd.fix", false)) {
+                    found.onUssdFinished(ussdMessage, isUssdRequest);
+                } else {
+                    found.onUssdRelease();
+                }
             } else if (isUssdError) {
                 found.onUssdFinishedError();
             } else {
diff --git a/src/java/com/android/internal/telephony/dataconnection/DcTracker.java b/src/java/com/android/internal/telephony/dataconnection/DcTracker.java
index 452b120..26a58d4 100644
--- a/src/java/com/android/internal/telephony/dataconnection/DcTracker.java
+++ b/src/java/com/android/internal/telephony/dataconnection/DcTracker.java
@@ -3344,11 +3344,12 @@ public class DcTracker extends Handler {
     }
 
     protected void setDataProfilesAsNeeded() {
+        boolean forceCognitive = SystemProperties.getBoolean("persist.sys.phh.radio.force_cognitive", false);
         if (DBG) log("setDataProfilesAsNeeded");
         if (mAllApnSettings != null && !mAllApnSettings.isEmpty()) {
             ArrayList<DataProfile> dps = new ArrayList<DataProfile>();
             for (ApnSetting apn : mAllApnSettings) {
-                if (apn.modemCognitive) {
+                if (apn.modemCognitive || forceCognitive) {
                     DataProfile dp = new DataProfile(apn);
                     if (!dps.contains(dp)) {
                         dps.add(dp);
-- 
2.7.4

