From b50a2caf452105d56035424ade2581c58a83c082 Mon Sep 17 00:00:00 2001
From: svoboda18 <ruspam18@gmail.com>
Date: Fri, 8 Feb 2019 17:44:47 +0000
Subject: [PATCH] Fix recent keys on mtk

Change-Id: Ia963688b8fb759ff9d944aa031fbe1fc748cdcc3

diff --git a/data/keyboards/Generic.kl b/data/keyboards/Generic.kl
index 8699cb49..20afca1b 100644
--- a/data/keyboards/Generic.kl
+++ b/data/keyboards/Generic.kl
@@ -146,7 +146,7 @@ key 123   EISU
 key 124   YEN
 key 125   META_LEFT
 key 126   META_RIGHT
-key 127   MENU
+key 127   APP_SWITCH
 key 128   MEDIA_STOP
 # key 129 "KEY_AGAIN"
 # key 130 "KEY_PROPS"
@@ -158,7 +158,7 @@ key 135   PASTE
 # key 136 "KEY_FIND"
 key 137   CUT
 # key 138 "KEY_HELP"
-key 139   MENU
+key 139   APP_SWITCH
 key 140   CALCULATOR
 # key 141 "KEY_SETUP"
 key 142   SLEEP
@@ -406,7 +406,7 @@ key 484   B                 FUNCTION
 
 key 522   STAR
 key 523   POUND
-key 580   APP_SWITCH
+key 580   MENU
 key 582   VOICE_ASSIST
 
 # Keys defined by HID usages
-- 
2.17.1

